http-api-data.cabal

```diff
@@ -47,6 +47,8 @@ library
                    , containers            >= 0.5.10.2 && < 0.7
                    , text                  >= 1.2.3.0  && < 1.3 || >=2.0 && <2.1
                    , transformers          >= 0.5.2.0  && < 0.7
+    if impl(ghc >= 9.4.1)
+    build-depends:   generically
```

Comment:
This looks wrong

Reply:
Pups yes. >= is the wrong way. Why did everything build though? :D

Reply:
CI is not run until I approve this PR, and I didn't bother. Please try locally with GHC-9.2 or older.

Reply:
It runs on my fork… But yeah the reason is probably that the check is ignored

Reply:
Seems like it’s fixed now :)